includes/class-wc-ajax.php

```diff
@@ -1076,14 +1076,24 @@ public static function add_coupon_discount() {
 		$response = array();
 
 		try {
-			$order_id = absint( $_POST['order_id'] );
-			$order    = wc_get_order( $order_id );
-			$result   = $order->apply_coupon( wc_clean( $_POST['coupon'] ) );
+			$order_id           = absint( $_POST['order_id'] );
+			$order              = wc_get_order( $order_id );
+			$calculate_tax_args = array(
+				'country'  => strtoupper( wc_clean( $_POST['country'] ) ),
+				'state'    => strtoupper( wc_clean( $_POST['state'] ) ),
+				'postcode' => strtoupper( wc_clean( $_POST['postcode'] ) ),
+				'city'     => strtoupper( wc_clean( $_POST['city'] ) ),
```

Comment:
Is there a reason not to unslash these? I can't match N'Djamena city specific tax as it gets sent to db as 
```
( locations2.location_type = 'city' AND locations2.location_code = 'N\\\'DJAMENA' )
```

(Same goes for the normal Recalculate button I think)

Reply:
This was resolved with the PHPCS merge.